src/components/ResourceHeader/index.jsx

```diff
@@ -32,32 +23,19 @@ const ResourceHeader = ({
   const md = useMediaQuery({ minWidth: 0, maxWidth: 768 });
   const { limit, offset, count, setLimit, setOffset } = resource;
   const intCount = count ? parseInt(count) : 0;
-  const [referenceElement, setReferenceElement] = useState(null);
-  const [popperElement, setPopperElement] = useState(null);
-  const [arrowElement, setArrowElement] = useState(null);
-  const { styles, attributes } = usePopper(referenceElement, popperElement, {
-    modifiers: [{ name: 'arrow', options: { element: arrowElement } }],
-  });
 
   return (
     <div className="dc-c-resource-header">
-      <div className="ds-l-row">
-        <div className="ds-l-col--12">
-          {includeFiltered && (
-            <Link className="ds-c-button ds-c-button--solid" to={`/dataset/${id}/data`}>
-              View and filter data
-            </Link>
-          )}
-        </div>
-      </div>
       <div className="ds-l-row ds-u-align-items--center">
         <div className="ds-l-col--12 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center">
           <div className="ds-u-font-weight--bold ds-u-margin-bottom--2">
```

Comment:
The `ds-u-margin-bottom--2` should probably move up to the parent container. When the results disappear during loading so does the margin bottom making the whole section move a little.